tests/unit/plugins/module_utils/test_healthchecksio.py

```diff
@@ -2,19 +2,61 @@
 
 __metaclass__ = type
 
-from argparse import ArgumentParser
-
 from ansible_collections.community.general.tests.unit.compat import unittest
-from ansible_collections.community.general.tests.unit.compat.mock import patch
-
-# from ansible.errors import AnsibleError
-from ansible.module_utils import six
+from ansible_collections.community.general.tests.unit.compat.mock import MagicMock
 
 from ansible_collections.community.healthchecksio.plugins.module_utils.healthchecksio import (
-    HealthchecksioHelper,
+    Response,
 )
 
 
-class TestHealthchecksioModuleUtil:
-    def test_healthchecksio(self):
-        assert True
+class TestResponseObject(unittest.TestCase):
+    def test_reads_repoonse_body(self):
+        resp = MagicMock()
+        resp.read = MagicMock()
+        resp.read.return_value = '{"foo":"bar"}'
+        info = {}
+        Response(resp, info)
+        self.assertEquals(resp.read.call_count, 1)
+
+    def test_json_decodes(self):
+        resp = MagicMock()
+        resp.read = MagicMock()
+        resp.read.return_value = '{"foo":"bar"}'
+        info = {}
+        r = Response(resp, info)
+        j = r.json
+        self.assertEquals(j, {"foo": "bar"})
```

Comment:
ansible-test seems to favour pytest. Would these be better as `assert j == {"foo": "bar"}` and remove the base class?